todo/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import Todos from './components/Todos';
```

Comment:
Make sure to comment these out since they will throw an error if the file is not created.